src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -3836,6 +3836,9 @@ gregtech.machine.item_bus.export.zpm.name=ZPM Output Bus
 gregtech.machine.item_bus.export.uv.name=UV Output Bus
 gregtech.machine.item_bus.export.uhv.name=UHV Output Bus
 
+gregtech.bus.collapse=Allow Bus to auto combine item stacks %s
```

Comment:
I would suggest two different messages, one for true and one for false. The wording on this combined messages is just a bit weird.

"Bus Allowed to combine items", "Bus Disallowed to combine items". Something like that